examples/experimental/ProductionMonitoring/stream_table_api.ipynb

```diff
@@ -0,0 +1,170 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "eccb44d3",
+   "metadata": {},
+   "source": [
+    "# Weave StreamTable Usage\n",
+    "\n",
+    "This notebook demonstrates basic Weave StreamTable usage with interactive examples.\n",
+    "\n",
+    "## Step 0: Setup\n",
+    "\n",
+    "All the StreamTables created in this notebook will be saved to the WB_PROJECT under the WB_ENTITY account on the public W&B cloud. \n",
+    "\n",
+    "**Please login to W&B and set your WB_ENTITY** before running this demo."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "51c30e7a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import weave\n",
+    "from weave.monitoring import StreamTable"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5684f0b8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "WB_ENTITY = \n",
+    "WB_PROJECT = \"mesa\"\n",
+    "STREAM_TABLE_NAME = \"my_stream_table\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f676d96",
+   "metadata": {},
+   "source": [
+    "## Step 1: Define a StreamTable\n",
+    "\n",
+    "StreamTable has a single required argument: the name of the StreamTable object.\n",
+    "\n",
+    "```python\n",
+    "st = StreamTable(\"stacey/mesa/my_stream_table\")\n",
+    "```\n",
+    "\n",
+    "This takes the form `my_wb_entity/my_wb_project_name/my_stream_table_name` where you can modify the component names to the relevant strings (e.g. your W&B username or shared W&B team name, a new or existing W&B project name).\n",
+    "\n",
+    "**Note**: if you are logged in on the current machine and have a ~/.netrc file saved with your W&B credentials, this will be the default entity used if you do not pass in an entity string component to the StreamTable constructor."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25bfa9dc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "st = StreamTable(f\"{WB_ENTITY}/{WB_PROJECT}/{STREAM_TABLE_NAME}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "073d25c9",
+   "metadata": {},
+   "source": [
+    "## Step 2: Log some data\n",
+    "\n",
+    "To add rows to the StreamTable, call `.log()` on the StreamTable object. \n",
+    "`.log()` accepts a single dictionary or a list of dictionaries, where each dictionary entry corresponds to one row of the table. In each dictionary, the keys are column names and the values are the corresponding cell values.\n",
```

Comment:
good opportunity to link to the weave types documentation here for a list of what types it accepts. This is where I think markdown/proper docs is better than notebooks since it generates more stable links

Reply:
sounds good going forward—perhaps we add StreamTable.md to a growing single markdown doc of types? right now that's TECHNICAL_OVERVIEW.md I think, which is still an open PR here: https://github.com/wandb/weave/pull/114